### PR TITLE
EVG-2052 Write migration job to copy tests to new collection

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: migrations/anser_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 
 func TestAnserBasicPlaceholder(t *testing.T) {
 	assert := assert.New(t)
-	app, err := Application(&mock.Environment{})
+	app, err := Application(&mock.Environment{}, "foo")
```
Comment (tychoish) [inline migrations/anser_test.go]: should probably use `mci_test`

File: migrations/testresults.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	URLRaw    string  `json:"url_raw" bson:"url_raw,omitempty"`
+	LogID     string  `json:"log_id,omitempty" bson:"log_id,omitempty"`
+	LineNum   int     `json:"line_num,omitempty" bson:"line_num,omitempty"`
+	ExitCode  int     `json:"exit_code" bson:"exit_code"`
+	StartTime float64 `json:"start" bson:"start"`
+	EndTime   float64 `json:"end" bson:"end"`
+
+	// Together, TaskID and Execution identify the task which created this testResult
+	TaskID    string `bson:"task_id" json:"task_id"`
+	Execution int    `bson:"task_execution" json:"task_execution"`
```
Comment (tychoish) [inline migrations/testresults.go]: probably don't need to tag the json?

I still think we could probably get away with putting this inside of the function scope.

File: migrations/anser_test.go (hunk 1 above)
Comment (bsamek) [reply migrations/anser_test.go]: done

File: migrations/testresults.go (hunk 2 above)
Comment (bsamek) [reply migrations/testresults.go]: done

**Pull request merged**